src/renderers/shaders/ShaderChunk/aomap_fragment.glsl.js

```diff
@@ -10,7 +10,14 @@ export default /* glsl */`
 
 		float dotNV = saturate( dot( geometryNormal, geometryViewDir ) );
 
-		reflectedLight.indirectSpecular *= computeSpecularOcclusion( dotNV, ambientOcclusion, material.roughness );
+		float specularOcclusion = computeSpecularOcclusion( dotNV, ambientOcclusion, material.roughness );
```

Comment:
This should use the clearcoat roughness instead, so I think you need to call the function twice.